Applications/Spire/Include/Spire/Charting/DropDownColorPicker.hpp

```diff
@@ -43,6 +44,7 @@ namespace Spire {
       QColor m_current_color;
       QColor m_stored_color;
       bool m_has_mouse;
+      std::unique_ptr<DropShadow> m_color_picker_drop_shadow;
```

Comment:
Remove use of ```unique_ptr```s for ```DropShadow``` in situations where a parent is provided.

Reply:
Should I keep passing a parent, or is it redundant?

Reply:
```DropShadow``` makes use of a parent so it's necessary.